.github/workflows/publish-release-type.yaml

```diff
@@ -66,3 +66,24 @@ jobs:
           channel: team-pitch-black
           color: red
           verbose: true
+      - name: get version
+        id: get_version
+        run: echo ::set-output name=VERSION::$(echo ${GITHUB_REF#refs/*/})
+      - name: slack on success - aws-dev
+        if: ${{ success() }}
+        uses: actions-ecosystem/action-slack-notifier@fc778468d09c43a6f4d1b8cccaca59766656996a
+        with:
+          slack_token: ${{ secrets.slackToken }}
+          message: ":tada: ${{ steps.get_version.outputs.VERSION }} released!"
+          channel: aws-dev
+          color: green
+          verbose: false
+      - name: slack on failure - aws-dev
+        if: ${{ failure() }}
+        uses: actions-ecosystem/action-slack-notifier@fc778468d09c43a6f4d1b8cccaca59766656996a
+        with:
+          slack_token: ${{ secrets.slackToken }}
+          message: ":ahhhhhhhhh: ${{ steps.get_version.outputs.VERSION }} has failed"
+          channel: aws-dev
+          color: red
+          verbose: true
```

Comment:
thinking out loud here but do we need to announce in aws channel if the releases failed? 🤔  it might raise more alarms 🙈 I think as long as our team channel knows releases failed then it's fine. Although I am not sure how it was setup before in circleci

Reply:
removed